platforms/core-configuration/file-collections/src/test/groovy/org/gradle/api/internal/file/SubtractingFileCollectionExecutionTimeValueTest.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.file
+
+
+import spock.lang.Specification
+
+import static org.gradle.api.internal.file.TestFiles.fileCollectionFactory
+
+class SubtractingFileCollectionExecutionTimeValueTest extends Specification {
```

Comment:
💅  The test class seems to be more about the `SubtractingFileCollection`. Testing execution time value behavior is only one aspect. I'd rename the test class to accommodate for future unit tests for the collection itself.

Reply:
We have a bunch of tests similarly named like this (by me). They are targeting only ETV intentionally, since further ETV implementations will not so simple, and perhaps will require more extensive coverage.